csharp/src/SeedLang/Interpreter/Compiler.cs

```diff
@@ -96,12 +78,64 @@ private struct ExpressionInfo {
     }
 
     protected override void Visit(ComparisonExpression comparison) {
-      throw new NotImplementedException();
+      // TODO: support multiple operators. Current implementation only supports two operators.
```

Comment:
What are the "two operators" that are currently supported? Should they actually be "registers" to hold values? What is the case that requires "multiple operators"? Can you show an example in the comments?

Reply:
Modified the comments and added examples.
There was typos in the comments. It shall be `multiple operands` and `two operands`.